frontend/styles/components/calendar/WeeklyView.module.scss

```
@@ -1,4 +1,5 @@
 @import 'https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap';
+@import '../../Variables.module.scss';
```

Stylelint (17.14.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)